modules/nextflow/src/main/groovy/nextflow/container/PodmanBuilder.groovy

```diff
@@ -102,17 +102,26 @@ class PodmanBuilder extends ContainerBuilder<PodmanBuilder> {
         result << makeVolumes(mounts)
         result << '-w "$PWD" '
 
-        if( entryPoint )
+        if (entryPoint)
             result << '--entrypoint ' << entryPoint << ' '
 
-        if( runOptions )
+        if (runOptions)
             result << runOptions.join(' ') << ' '
 
+        if (cpus) {
+            result << "--cpu-quota ${cpus} "
+        }
```

Comment:
Not a podman expert, but I would expect it has the same options of docker, that are 

```
      --cpu-quota int                  Limit CPU CFS (Completely Fair Scheduler) quota
      --cpu-rt-period int              Limit CPU real-time period in microseconds
      --cpu-rt-runtime int             Limit CPU real-time runtime in microseconds
  -c, --cpu-shares int                 CPU shares (relative weight)
      --cpus decimal                   Number of CPUs
```

Reply:
Yup, you are right. I simply opted for the base option here rather than the alias http://docs.podman.io/en/latest/markdown/podman-run.1.html#cpu-quota-limit

<img width="731" alt="image" src="https://user-images.githubusercontent.com/12799326/102212605-80969e00-3efb-11eb-9a04-272b477174fd.png">